src/main/java/net/sf/rails/algorithms/RevenueCalculatorMulti.java

```diff
@@ -154,38 +154,38 @@ final protected void runBottom(final int trainId) {
             returnEdge(trainId, edgeId);
             trainStackPos[trainId]--; // pull from stack
         }
-        
+
 //        trainStackPos[trainId]--; // pull from stack
         trainBottomActive[trainId] = false;
-        log.debug("RC: finished bottom of " + trainId);
+        log.debug("RC: finished bottom of {}", trainId);
 
 
     }
 
     private final void nextVertex(final int trainId, final int vertexId) {
 
         // 1. encounterVertex adds value and returns true if value vertex
-        Terminated trainTerminated = Terminated.NotYet; 
+        Terminated trainTerminated = Terminated.NOT_YET;
```

Comment:
Which Styleguide are you using for this rename?

Reply:
Constants being uppercase is a pretty standard convention-- but in this case to answer your question specifically, in this case it's part of the standard checkstyle rules and out-of-the-box SonarCloud/SonarAnalyzer rule sets. 

But even more so, the code base already had a mix of upper case constants and lower or mixed case so in reality the change here isn't so much about a particular style guide but to bring a standard into our own code and given that the bulk of the constant declarations were already upper case it make sense to continue with that approach.

Reply:
I'm fine with this change, but in my opinion `_` as a divider between words looks strange in Java. I know this pattern mostly from C/C++

Reply:
Yep, understood. Sometimes I really miss writing C & C++ :)